src/perception/tag_detector/tag_detector.hpp

```diff
@@ -4,6 +4,10 @@
 
 #include <opencv2/aruco.hpp>
 #include <opencv2/core/mat.hpp>
+#include <opencv2/imgproc.hpp>
+
+#include <opencv2/core/types.hpp>
+#include <opencv2/highgui/highgui.hpp>
```

Comment:
Remove

Reply:
Looks like opencv is actually included twice, remove one of the groups and also remove highgui since we are using ROS to view images